src/screens/Donate/Donate.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Fragment } from 'react';
+import { Grid } from '@material-ui/core';
+import { Icon } from '@iconify/react';
+import testTube from '@iconify/icons-twemoji/test-tube';
+import loveLetter from '@iconify/icons-twemoji/love-letter';
+import deliveryTruck from '@iconify/icons-twemoji/delivery-truck';
+import {
+  EmbedWrapper,
+  ContentWrapper,
+  SectionHeader,
+  ContentSection,
+  ListHeader,
+  BodyCopy,
+  ListSection,
+} from 'screens/Donate/Donate.style';
+import PartnersSection from 'screens/HomePage/PartnersSection';
+
+const roadmapListContent = [
+  {
+    header: 'Improve our data quality',
+    listItems: [
+      'More accurate data',
+      'More transparency of data sources',
+      'Encourage states to adopt better contact tracing data practices',
+    ],
+  },
+  {
+    header: 'Inspire more action',
+    listItems: [
+      'Localized recommendations for masking, schools, and businesses based on new daily cases and positivity rate',
+      'Localized masking compliance; illustrate impact of increasing compliance',
+      'Continually survey our audience to understand their biggest needs as they change',
+    ],
+  },
+  {
+    header: 'Expand our reach',
+    listItems: [
+      'More educational content so that more people can understand our Warning System',
+      'Closer relationships with high-risk and underserved communities',
+      'More tools for scientists and journalists to help uncover new trends and correlations',
+    ],
+  },
+];
+
+const Donate = () => {
+  return (
+    <Fragment>
+      <EmbedWrapper>
+        <iframe
+          title="Donation embed"
+          src="https://givebutter.com/embed/c/covidactnow"
+          width="100%"
+          height="100%"
+          style={{ maxWidth: '601px' }}
+          name="givebutter"
+          frameBorder="0"
+          scrolling="auto"
+          seamless
+          data-allowPaymentRequest
+        ></iframe>
+        <script src="https://givebutter.com/js/widget.js"></script>
+      </EmbedWrapper>
+      <ContentWrapper container justify="center">
+        <Grid item xs={12} sm={6}>
+          <ContentSection>
+            <Icon icon={loveLetter} />
+            <SectionHeader>How your donation helps</SectionHeader>
```

Comment:
Consider using `h1` and `h2` components for the headers instead of styled paragraphs. Assistive technologies such as screen readers rely on the structure of the page (some screen readers have keyboard shortcuts to navigate headers). See https://web.dev/headings-and-landmarks/

The `Typography` component can have a style (let's say `h4`) while using a different component (`h1`) which makes styling a bit easier. See [About.tsx#L54](https://github.com/covid-projections/covid-projections/blob/0074098e32a22d2e75ef53e054866947329b79c1/src/screens/About/About.tsx#L54) for an example of how to use semantic headers.

I think that the main "title" on the page is the embed title, so you probably want to use `h2` for "How your donation helps" and `h3` for "Improve our Data Quality".

Note: We don't use the correct tags in many places on our site, I would like to fix it eventually. We can skip this if we need to ship this quickly.

Reply:
great note. going to be mindful of this going forward.